projects/kit/components/input-tag/input-tag.component.ts

```diff
@@ -127,6 +138,9 @@ export class TuiInputTagComponent
     @tuiDefaultProp()
     expandable = true;
 
+    @Input()
+    rows = Infinity;
```

Comment:
I wonder if we can deprecate `expandable` in favor of this input with 1 === expandable: false.

Reply:
Scrolling fixed. 
For now I can't deprecate `expandable` due to backward compatibility. I added TODO for 4.0